app/helpers/transactions_helper.rb

```diff
@@ -0,0 +1,5 @@
+module TransactionsHelper
+  def category_options
+    options_for_select(current_user.categories.map { |category| [category.name, category.id] }) || []
+  end
```

Comment:
@sinpai looks like a huge line. Let's split it too several:
```ruby
def category_options
  options_for_select(categories_by_name_and_id) || []
end

private

def categories_by_name_and_id
  current_user.categories.map { |category| [category.name, category.id] }
end
```

Reply:
@sinpai why do we need `[]` here? wouldn't `options_for_select([])` return empty array?